src/test/java/org/folio/rest/api/PrecedingSucceedingTitleTest.java

```diff
@@ -0,0 +1,297 @@
+package org.folio.rest.api;
+
+import static org.folio.rest.api.TestBaseWithInventoryUtil.UUID_ISBN;
+import static org.folio.rest.api.TestBaseWithInventoryUtil.identifier;
+import static org.folio.rest.support.http.InterfaceUrls.holdingsStorageUrl;
+import static org.folio.rest.support.http.InterfaceUrls.instancesStorageUrl;
+import static org.folio.rest.support.http.InterfaceUrls.itemsStorageUrl;
+import static org.folio.rest.support.http.InterfaceUrls.precedingSucceedingTitleUrl;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.api.entities.Instance;
+import org.folio.rest.api.entities.PrecedingSucceedingTitle;
+import org.folio.rest.jaxrs.model.Errors;
+import org.folio.rest.support.Response;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class PrecedingSucceedingTitleTest extends ResourceTestBase {
```

Comment:
Could you please make sure that following edge case is covered as well:
* PUT/DELETE/GET fails with error when specified wrong id;